R/moduleCoverage.R

```diff
@@ -47,33 +56,37 @@ setGeneric("moduleCoverage", function(name, path) {
 #' @rdname moduleCoverage
 setMethod(
   "moduleCoverage",
-  signature(name = "character", path = "character"),
-  definition = function(name, path) {
-    tmpdir <- file.path(tempdir(), "moduleCoverage")
-    dir.create(tmpdir); on.exit(unlink(tmpdir, recursive = TRUE))
-    fnDir <- file.path(path, name, "moduleFunctions") %>%
+  signature(name = "character", path = "character", byFunctionName = "logical"),
+  definition = function(name, path, byFunctionName) {
+    tmpdir <- tempdir(); on.exit(unlink(tmpdir, recursive = TRUE))
```

Comment:
Don't delete the entire per-session temp dir because other things may rely on it. You need to create a subdir inside of `tmpdir()`, *i.e.*:

```r
tmpdir <- file.path(tempdir(), "moduleCoverage") %>% checkPath(create = TRUE)
on.exit(unlink(tmpdir, recursive = TRUE))
```